### ISSUE TITLE
api/timesheets does not return hourly rate, fixed rate

### ISSUE THREAD

**Issue by Filechaser** [kevinpapst/kimai2#2743]
api/timesheets does not return hourly rate and fixed rate fields.
api/timesheet/id does.

Bug?

Thanks for your help

**Comment by kevinpapst** [comment]
Yes,  on purpose. Each additional field increases the content length and not every user is allowed to see the fields.  I  am not near a computer to check:  isn't there a "full" flag to return more fields in the response?

**Comment by Filechaser** [comment]
Yes there is a full flag.
The Full flag includes the referenced sub-tables. 
Unfortunately the fields hourlyRate and fixedRate are not included despite the "full"-flag.
Could you include them in the next release when using full?

**Comment by kevinpapst** [comment]
These fields can only be seen by users with some permissions. This needs to be reflected in the API.

Nobody ever asked for these fields, so it doesn't have the biggest priority. But if you want to provide a PR, I'd be happy to review.

**Comment by rahulkrishnans** [comment]
In the older version (1.10.2) it is available in user level configuration, but in the version 1.18.2, it is not available. If it is available, it is very much useful to us. Can you please check.

Also when I try to update fixedRate cell, it always shows "This form should not contain extra fields.", but I can patch the other meta fields as in 1.10.2. Is this also an issue related to the above?

```
{
  "code": 400,
  "message": "Validation Failed",
  "errors": {
    "errors": [
      "This form should not contain extra fields."
    ],
    "children": {
      "project": {},
      "activity": {},
      "description": {},
      "user": {},
      "exported": {},
      "billable": {},
      "tags": {}
    }
  }
}
```

API used:
`curl -k -X PATCH "http://kimai.org/api/timesheets/ID" -H "accept: application/json" -H "X-AUTH-USER: USERNAME" -H "X-AUTH-TOKEN: PASSWORD" -H "Content-Type: application/json" -d "{ \"fixedRate\": 100,  \"hourlyRate\": 0}"`
